SQL/0000-00-00-schema.sql

```diff
@@ -555,6 +555,7 @@ CREATE TABLE `flag` (
   `Flag_status` enum('P','Y','N','F') default NULL,
   `UserID` varchar(255) default NULL,
   `Testdate` timestamp NOT NULL default CURRENT_TIMESTAMP on update CURRENT_TIMESTAMP,
+  `Data` TEXT default NULL,
```

Comment:
Can we use the native JSON data type? 
https://dev.mysql.com/doc/refman/5.7/en/json.html